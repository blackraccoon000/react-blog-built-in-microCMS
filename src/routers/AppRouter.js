import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import AppRoutes from './AppRoutes';

const AppRouter = () => {
  console.log('router called');
  return (
    <BrowserRouter>
      <AppRoutes />
    </BrowserRouter>
  );
};

export default AppRouter;

// http://localhost:8082/article/dist/assets/favicon.ico
// http://localhost:8082/article/dist/bundle.js?7b667c5338b3b98a24f8
// http://localhost:8082/dist/assets/android-chrome-144x144.png
