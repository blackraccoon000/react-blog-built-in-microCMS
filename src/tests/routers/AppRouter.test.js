import React from 'react';
import { render } from '@testing-library/react';
import AppRouter from '../../routers/AppRouter';

/**
 * use manual mock
 */
// jest.mock("../../components/Header")
// jest.mock("../../components/Home")

it('AppRouter経由でHomeディレクトリが描画されることを確認', () => {
  const wrapper = render(<AppRouter />);
  expect(wrapper.container).toMatchSnapshot();
});
