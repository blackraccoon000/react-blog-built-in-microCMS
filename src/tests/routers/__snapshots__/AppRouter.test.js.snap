// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppRouter経由でHomeディレクトリが描画されることを確認 1`] = `
<div>
  <header
    class="HeaderContainer__Header-f510c1-0 dZqlpm"
  >
    <div
      class="HeaderContainer__Container-f510c1-1 eRkSIU"
    >
      <div
        class="HeaderStart__StartDiv-sc-9w4ojd-0 jXaUJI"
      >
        <div
          class="HeaderStart__HeaderItem-sc-9w4ojd-1 eaGzUa"
        >
          <a
            class="HeaderStart__HeaderLink-sc-9w4ojd-2 bGTXxb"
            href="#"
          />
          <svg
            class="IconRac__Svg-sc-1pijcu6-0"
            color="#8b8c8e"
            height="45"
            size="45"
            version="1.1"
            viewBox="0 0 30 30"
            width="45"
            x="0"
            xml:space="preserve"
            xmlns="http://www.w3.org/2000/svg"
            xmlns:xlink="http://www.w3.org/1999/xlink"
            y="0"
          >
            <g>
              <path
                class="IconRac__Path-sc-1pijcu6-1"
                color="#8b8c8e"
                d="M25.4,13c-0.43-1.69-1.06-3.3-1.87-4.81C23.36,7.87,23.18,7.55,23,7.24c-0.46,0.35-0.9,0.72-1.33,1.11 c-0.82,0.75-1.59,1.56-2.29,2.43c-0.24,0.3-0.48,0.62-0.71,0.93c0.39,0.07,0.78,0.15,1.17,0.24c1.34,0.32,2.64,0.76,3.87,1.31 c0.68,0.31,1.34,0.65,1.99,1.02C25.61,13.85,25.51,13.42,25.4,13z M20.57,11.06c0.51-0.6,1.06-1.17,1.64-1.71 c0.43,0.85,0.8,1.73,1.11,2.64C22.43,11.62,21.51,11.3,20.57,11.06z"
                fill="#8b8c8e"
                size="45"
              />
              <path
                class="IconRac__Path-sc-1pijcu6-1"
                color="#8b8c8e"
                d="M26.06,18.28c0-0.81-0.05-1.61-0.13-2.4c-0.03-0.24-0.17-0.45-0.37-0.57c-0.5-0.29-1.02-0.56-1.55-0.81 c-1.53-0.72-3.15-1.26-4.85-1.6c-0.38-0.08-0.77-0.14-1.15-0.2c-0.92,1.45-1.68,3.01-2.23,4.66c0,0,0,0,0,0 c0.32,0.1,0.63,0.21,0.93,0.34c1.26,0.55,2.35,1.41,3.18,2.49c0.08,0.1,0.22,0.13,0.33,0.06c0.59-0.37,1.22-0.69,1.87-0.96 c0.04-0.02,0.09-0.01,0.13,0.02c0.73,0.71,1.32,1.56,1.74,2.51c0.07,0.15,0.3,0.12,0.32-0.04c0.13-0.94,0.19-1.89,0.19-2.86 c0-0.12,0-0.24,0-0.36c0.52-0.11,1.06-0.19,1.6-0.23C26.06,18.31,26.06,18.29,26.06,18.28z M20.24,17.91 c-0.37,0-0.69-0.17-0.91-0.43c-0.24-0.28-0.35-0.68-0.23-1.09c0.1-0.39,0.43-0.72,0.82-0.82c0.8-0.21,1.52,0.38,1.52,1.15 c0,0.64-0.51,1.17-1.15,1.19C20.27,17.91,20.25,17.91,20.24,17.91z"
                fill="#8b8c8e"
                size="45"
              />
              <path
                class="IconRac__Path-sc-1pijcu6-1"
                color="#8b8c8e"
                d="M15.79,21.11c-0.16-0.13-0.35-0.23-0.56-0.26c-0.07-0.01-0.14-0.02-0.21-0.02c-0.03,0-0.07,0-0.1,0 c-0.17,0.01-0.34,0.07-0.48,0.15c-0.36,0.2-0.61,0.59-0.61,1.04c0,0.54,0.36,1,0.85,1.14c0.01,0,0.02,0,0.02,0.01 c0.1,0.03,0.2,0.04,0.31,0.04c0.14,0,0.27-0.02,0.39-0.07c0.04-0.01,0.08-0.03,0.11-0.05c0.4-0.19,0.68-0.6,0.68-1.08 C16.21,21.66,16.05,21.33,15.79,21.11z"
                fill="#8b8c8e"
                size="45"
              />
              <path
                class="IconRac__Path-sc-1pijcu6-1"
                color="#8b8c8e"
                d="M14.37,17.08c-0.51-1.57-1.2-3.05-2.04-4.44c-0.4,0.05-0.79,0.11-1.17,0.18c-1.87,0.33-3.66,0.91-5.33,1.7 c-0.6,0.28-1.18,0.59-1.74,0.92c-0.11,0.93-0.16,1.88-0.14,2.85c0,0,0,0,0,0c0.49,0.01,0.97,0.05,1.44,0.12 c0,0.17-0.01,0.34-0.01,0.52c0,0.98,0.07,1.94,0.19,2.89c0.03,0.21,0.31,0.25,0.4,0.06c0.46-1.04,1.14-1.97,1.97-2.72 c0.06-0.05,0.15-0.07,0.22-0.04c0.64,0.24,1.25,0.52,1.84,0.85c0.08,0.04,0.18,0.03,0.24-0.04c0.87-1.05,2.01-1.87,3.3-2.37 c0.29-0.11,0.59-0.21,0.89-0.29C14.4,17.19,14.39,17.13,14.37,17.08z M11.11,17.36c-0.22,0.29-0.56,0.47-0.95,0.47 c-0.08,0-0.16-0.01-0.23-0.02c-0.64-0.12-1.1-0.76-0.93-1.47c0.1-0.41,0.43-0.75,0.84-0.85c0.8-0.21,1.51,0.39,1.51,1.15 C11.35,16.91,11.26,17.15,11.11,17.36z"
                fill="#8b8c8e"
                size="45"
              />
              <path
                class="IconRac__Path-sc-1pijcu6-1"
                color="#8b8c8e"
                d="M11.02,10.74c-0.78-1.02-1.65-1.97-2.6-2.83C7.99,7.52,7.55,7.14,7.08,6.79c-0.21,0.34-0.41,0.68-0.6,1.03 c-0.9,1.66-1.58,3.46-2.01,5.35c-0.08,0.35-0.15,0.7-0.21,1.06c0.6-0.35,1.22-0.67,1.86-0.96c1.38-0.62,2.83-1.1,4.34-1.43 c0.4-0.08,0.8-0.16,1.21-0.22C11.46,11.33,11.25,11.03,11.02,10.74z M6.46,12.17c0.37-1.12,0.83-2.19,1.37-3.22 c0.7,0.64,1.36,1.33,1.97,2.06C8.64,11.31,7.53,11.7,6.46,12.17z"
                fill="#8b8c8e"
                size="45"
              />
            </g>
            <circle
              class="IconRac__Circle-sc-1pijcu6-2"
              color="#8b8c8e"
              cx="15"
              cy="15"
              fill="none"
              r="13.5"
              size="45"
              stroke="#8b8c8e"
              stroke-miterlimit="10"
            />
          </svg>
        </div>
        <details
          class="SummaryDetails__Details-yvm3jk-0 FmnlR"
        >
          <summary
            class="SummaryDetails__Summary-yvm3jk-1 bvzJpD"
          >
            Details
            <svg
              class="IconChevonDown__Svg-bi15uh-0 efyWSq"
              color="#8b8c8e"
              height="8"
              version="1.1"
              viewBox="0 0 15 8"
              width="15"
              x="0"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0"
            >
              <g>
                <path
                  class="IconChevonDown__Path-bi15uh-1"
                  color="#8b8c8e"
                  d="M7.5,7.83c-0.26,0-0.51-0.1-0.71-0.29L1.14,1.88c-0.39-0.39-0.39-1.02,0-1.41s1.02-0.39,1.41,0L7.5,5.41l4.95-4.95 c0.39-0.39,1.02-0.39,1.41,0s0.39,1.02,0,1.41L8.21,7.54C8.01,7.73,7.76,7.83,7.5,7.83z"
                  fill="#8b8c8e"
                  height="8"
                  width="15"
                />
              </g>
            </svg>
          </summary>
          <div
            class="SummaryDetails__DropdownMenu-yvm3jk-3 dabUSc"
          >
            <a
              class="SummaryDetails__OtherLinks-yvm3jk-2 bXCcbq"
              href="#"
            >
              Link α
            </a>
            <a
              class="SummaryDetails__OtherLinks-yvm3jk-2 bXCcbq"
              href="#"
            >
              Link β
            </a>
            <a
              class="SummaryDetails__OtherLinks-yvm3jk-2 bXCcbq"
              href="#"
            >
              Link γ
            </a>
          </div>
        </details>
      </div>
      <div
        class="HeaderEnd__EndDiv-sc-1esdvrb-0 gBcvVU"
      >
        <form
          class="SearchForm__Form-y6g7a-0 gZfagT"
        >
          <label
            class="SearchForm__Label-y6g7a-1 hpgKvy"
            for="search-form"
          >
            <svg
              class="IconFocus__Svg-sc-14mfl82-0 dVifBr SearchForm__Focus-y6g7a-4 btwUqH"
              color="#8b8c8e"
              height="20"
              size="20"
              version="1.1"
              viewBox="0 0 20 20"
              width="20"
              x="0"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0"
            >
              <g>
                <path
                  class="IconFocus__Path-sc-14mfl82-1 SearchForm__Focus-y6g7a-4 btwUqH"
                  color="#8b8c8e"
                  d="M15.5 14h-.79l-.28-.27A6.471 6.471 0 0 0 16 9.5 6.5 6.5 0 1 0 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                  fill="#8b8c8e"
                  size="20"
                />
              </g>
            </svg>
            <input
              class="SearchForm__Input-y6g7a-2 grUhWi"
              id="search-form"
              name="q"
              placeholder="Search or jump to…"
              size="10"
              type="text"
            />
            <svg
              class="IconSearchKeySlash__Svg-lcn4uw-0 faSfIx SearchForm__SearchKeySlash-y6g7a-3 dLIVhZ"
              color="#8b8c8e"
              height="20"
              size="20"
              version="1.1"
              viewBox="0 0 20 20"
              width="20"
              x="0"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0"
            >
              <g>
                <path
                  class="IconSearchKeySlash__Path-lcn4uw-1 SearchForm__SearchKeySlash-y6g7a-3 dLIVhZ"
                  color="#8b8c8e"
                  d="M2.5,0.5C2.22,0.5,2,0.73,2,1v18c0,0.27,0.22,0.5,0.5,0.5h15c0.27,0,0.5-0.23,0.5-0.5V1 c0-0.27-0.23-0.5-0.5-0.5H2.5z M16,17.5c0,0.27-0.23,0.5-0.5,0.5h-11C4.22,18,4,17.77,4,17.5v-15C4,2.22,4.22,2,4.5,2h11 C15.77,2,16,2.22,16,2.5V17.5z"
                  fill="#8b8c8e"
                  size="20"
                />
              </g>
              <polygon
                class="IconSearchKeySlash__Polygon-lcn4uw-2 SearchForm__SearchKeySlash-y6g7a-3 dLIVhZ"
                color="#8b8c8e"
                fill="#8b8c8e"
                points="8.6,17 6.73,17 11.4,3 13.27,3 "
                size="20"
              />
            </svg>
          </label>
        </form>
        <div
          class="NavItem__NavDiv-w5n2nb-0 bUcBMn"
          color="#8b8c8e"
          value="Sign In"
        >
          <a
            class="NavItem__NavA-w5n2nb-1 bICSxq"
            color="#8b8c8e"
            href="#"
            style="border: 2px solid #8b8c8e; color: rgb(139, 140, 142);"
            value="Sign In"
          >
            <span>
              Sign In
            </span>
          </a>
        </div>
        <div
          class="NavItem__NavDiv-w5n2nb-0 bUcBMn"
          color="#8b8c8e"
          value="Sign up"
        >
          <a
            class="NavItem__NavA-w5n2nb-1 bICSxq"
            color="#8b8c8e"
            href="#"
            style="border: 2px solid #8b8c8e; color: rgb(139, 140, 142);"
            value="Sign up"
          >
            <span>
              Sign up
            </span>
          </a>
        </div>
      </div>
    </div>
  </header>
  <div
    class="Home__Wrapper-sc-1yza5ij-0 eETUCZ"
  >
    <div
      class="Home__Container-sc-1yza5ij-1 eRfCXO"
    >
      <h1
        class="Home__Heading-sc-1yza5ij-2 wJefj"
      >
        Home Components
      </h1>
      <ul
        class="Home__Unordered-sc-1yza5ij-3"
      >
        <li
          class="Home__ListItem-sc-1yza5ij-4 jgCtOG"
        >
          first blog
           / 
          1 create blog now one
        </li>
        <li
          class="Home__ListItem-sc-1yza5ij-4 jgCtOG"
        >
          second blog
           / 
          2 create blog now two
        </li>
      </ul>
    </div>
  </div>
  <header
    class="FooterContainer__Footer-sc-13ni88s-0 eQPSMn"
  >
    <div
      class="FooterContainer__Container-sc-13ni88s-1 bBVYTd"
    >
      <ul
        class="FooterItems__UnorderedList-x03trw-1 bvfaai"
      >
        <li
          class="FooterItems__ListItem-x03trw-2 fhrBqN"
        >
          <a
            class="FooterItems__Link-x03trw-0 ivKumG"
            href="#"
          >
            © 2021 PlayWell,Inc
          </a>
        </li>
        <li
          class="FooterItems__ListItem-x03trw-2 fhrBqN"
        >
          <a
            class="FooterItems__Link-x03trw-0 ivKumG"
            href="#"
          >
            Terms
          </a>
        </li>
        <li
          class="FooterItems__ListItem-x03trw-2 fhrBqN"
        >
          <a
            class="FooterItems__Link-x03trw-0 ivKumG"
            href="#"
          >
            Privacy
          </a>
        </li>
        <li
          class="FooterItems__ListItem-x03trw-2 fhrBqN"
        >
          <a
            class="FooterItems__Link-x03trw-0 ivKumG"
            href="#"
          >
            Site Map
          </a>
        </li>
        <li
          class="FooterItems__ListItem-x03trw-2 fhrBqN"
        >
          <a
            class="FooterItems__Link-x03trw-0 ivKumG"
            href="#"
          >
            What is PlayWell?
          </a>
        </li>
      </ul>
      <ul
        class="FooterIcons__UnorderedList-k0zdql-0 llAHyc"
      >
        <li
          class="FooterIcons__ListItem-k0zdql-1 eABfIL"
        >
          <a
            href="https://qiita.com/whiteraccoon"
          >
            <div
              class="FooterIcons__QiitaBorder-k0zdql-2 khpikI"
              color="#586069"
              style="border: 2px solid #586069;"
            >
              <svg
                class="IconQiita__Svg-sc-15q13e2-0 dbNwHk"
                color="#586069"
                height="12"
                version="1.1"
                viewBox="0 0 426.57 130"
                width="34"
                x="0"
                xml:space="preserve"
                xmlns="http://www.w3.org/2000/svg"
                xmlns:xlink="http://www.w3.org/1999/xlink"
                y="0"
              >
                <g>
                  <circle
                    class="IconQiita__Circle-sc-15q13e2-2"
                    color="#586069"
                    cx="167.08"
                    cy="21.4"
                    fill="#586069"
                    height="12"
                    r="12.28"
                    width="34"
                  />
                  <path
                    class="IconQiita__Path-sc-15q13e2-1"
                    color="#586069"
                    d="M250.81 29.66h23.48v18.9h-23.48z"
                    fill="#586069"
                    height="12"
                    width="34"
                  />
                  <path
                    class="IconQiita__Path-sc-15q13e2-1"
                    color="#586069"
                    d="M300.76 105.26a22.23 22.23 0 01-6.26-.86 12.68 12.68 0 01-5.17-3 14.41 14.41 0 01-3.56-5.76 28 28 0 01-1.3-9.22V48.56h29.61v-18.9h-29.52V3.29h-20.17v83.34q0 11.16 2.83 18.27a27.71 27.71 0 007.7 11.2 26.86 26.86 0 0011.43 5.62 47.56 47.56 0 0012.34 1.53h15.16v-18zM0 61.7a58.6 58.6 0 015-24.21A62.26 62.26 0 0118.73 17.9 63.72 63.72 0 0139 4.78 64.93 64.93 0 0164 0a65 65 0 0124.85 4.78 64.24 64.24 0 0120.38 13.12A62 62 0 01123 37.49a58.6 58.6 0 015 24.21 58.34 58.34 0 01-4 21.46 62.8 62.8 0 01-10.91 18.16l11.1 11.1a10.3 10.3 0 010 14.52 10.29 10.29 0 01-14.64 0l-12.22-12.41a65 65 0 01-15.78 6.65 66.32 66.32 0 01-17.55 2.3 64.63 64.63 0 01-45.23-18A62.82 62.82 0 015 85.81 58.3 58.3 0 010 61.7zm21.64.08a43.13 43.13 0 0012.42 30.63 42.23 42.23 0 0013.43 9.09A41.31 41.31 0 0064 104.8a42 42 0 0030-12.39 42.37 42.37 0 009-13.64 43.43 43.43 0 003.3-17 43.77 43.77 0 00-3.3-17A41.7 41.7 0 0080.55 22 41.78 41.78 0 0064 18.68 41.31 41.31 0 0047.49 22a42.37 42.37 0 00-13.43 9.08 43.37 43.37 0 00-12.42 30.7zM331.89 78a47.59 47.59 0 013.3-17.73 43.22 43.22 0 019.34-14.47A44.25 44.25 0 01359 36a47.82 47.82 0 0118.81-3.58 42.72 42.72 0 019.26 1 46.5 46.5 0 018.22 2.58 40 40 0 017 3.84 44.39 44.39 0 015.71 4.63l1.22-9.47h17.35v85.83h-17.35l-1.17-9.42a42.54 42.54 0 01-5.84 4.67 43.11 43.11 0 01-7 3.79 44.86 44.86 0 01-8.17 2.59 43 43 0 01-9.22 1A47.94 47.94 0 01359 119.9a43.3 43.3 0 01-14.47-9.71 44.17 44.17 0 01-9.34-14.47 47 47 0 01-3.3-17.72zm20.27-.08a29.16 29.16 0 002.17 11.34 27 27 0 005.92 8.88 26.69 26.69 0 008.76 5.76 29.19 29.19 0 0021.44 0 26.11 26.11 0 008.72-5.76 27.57 27.57 0 005.88-8.84 29 29 0 002.16-11.38 28.62 28.62 0 00-2.16-11.22 26.57 26.57 0 00-5.93-8.8 27.68 27.68 0 00-19.51-7.9 28.29 28.29 0 00-10.77 2.05 26.19 26.19 0 00-8.71 5.75 27.08 27.08 0 00-5.84 8.8 28.94 28.94 0 00-2.13 11.31zm-194.97-30.5h19.78v73.54h-19.78zm49.25 0h19.78v73.54h-19.78z"
                    fill="#586069"
                    height="12"
                    width="34"
                  />
                  <circle
                    class="IconQiita__Circle-sc-15q13e2-2"
                    color="#586069"
                    cx="216.33"
                    cy="21.4"
                    fill="#586069"
                    height="12"
                    r="12.28"
                    width="34"
                  />
                </g>
              </svg>
            </div>
          </a>
        </li>
        <li
          class="FooterIcons__ListItem-k0zdql-1 eABfIL"
        >
          <a
            href="https://twitter.com/yutakaf_0326"
          >
            <svg
              class="IconTwitter__Svg-ig5tf0-0 kixtYN"
              color="#586069"
              height="20"
              version="1.1"
              viewBox="0 0 273.5 222.3"
              width="20"
              x="0"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0"
            >
              <g>
                <path
                  class="IconTwitter__Path-ig5tf0-1"
                  color="#586069"
                  d="M273.5 26.3a109.77 109.77 0 0 1-32.2 8.8 56.07 56.07 0 0 0 24.7-31 113.39 113.39 0 0 1-35.7 13.6 56.1 56.1 0 0 0-97 38.4 54 54 0 0 0 1.5 12.8A159.68 159.68 0 0 1 19.1 10.3a56.12 56.12 0 0 0 17.4 74.9 56.06 56.06 0 0 1-25.4-7v.7a56.11 56.11 0 0 0 45 55 55.65 55.65 0 0 1-14.8 2 62.39 62.39 0 0 1-10.6-1 56.24 56.24 0 0 0 52.4 39 112.87 112.87 0 0 1-69.7 24 119 119 0 0 1-13.4-.8 158.83 158.83 0 0 0 86 25.2c103.2 0 159.6-85.5 159.6-159.6 0-2.4-.1-4.9-.2-7.3a114.25 114.25 0 0 0 28.1-29.1"
                  fill="#586069"
                  fill-rule="evenodd"
                  height="20"
                  width="20"
                />
              </g>
            </svg>
          </a>
        </li>
        <li
          class="FooterIcons__ListItem-k0zdql-1 eABfIL"
        >
          <a
            href="https://www.facebook.com/yutaka.fujii.3979/"
          >
            <svg
              class="IconFacebook__Svg-xnxxqf-0 hpSlGI"
              color="#586069"
              height="18"
              version="1.1"
              viewBox="0 0 15.3 15.4"
              width="18"
              x="0"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0"
            >
              <g>
                <path
                  class="IconFacebook__Path-xnxxqf-1"
                  color="#586069"
                  d="M14.5 0H.8a.88.88 0 0 0-.8.9v13.6a.88.88 0 0 0 .8.9h7.3v-6h-2V7.1h2V5.4a2.87 2.87 0 0 1 2.5-3.1h.5a10.87 10.87 0 0 1 1.8.1v2.1h-1.3c-1 0-1.1.5-1.1 1.1v1.5h2.3l-.3 2.3h-2v5.9h3.9a.88.88 0 0 0 .9-.8V.8a.86.86 0 0 0-.8-.8z"
                  fill="#586069"
                  fill-rule="evenodd"
                  height="18"
                  width="18"
                />
              </g>
            </svg>
          </a>
        </li>
        <li
          class="FooterIcons__ListItem-k0zdql-1 eABfIL"
        >
          <a
            href="https://github.com/blackraccoon000"
          >
            <svg
              class="IconGitOct__Svg-sc-1lx6gqb-0 laMKJQ"
              color="#586069"
              height="22"
              version="1.1"
              viewBox="0 0 16 16"
              width="22"
              x="0"
              xml:space="preserve"
              xmlns="http://www.w3.org/2000/svg"
              xmlns:xlink="http://www.w3.org/1999/xlink"
              y="0"
            >
              <g>
                <path
                  class="IconGitOct__Path-sc-1lx6gqb-1"
                  color="#586069"
                  d="M8 0C3.58 0 0 3.58 0 8c0 3.54 2.29 6.53 5.47 7.59.4.07.55-.17.55-.38 0-.19-.01-.82-.01-1.49-2.01.37-2.53-.49-2.69-.94-.09-.23-.48-.94-.82-1.13-.28-.15-.68-.52-.01-.53.63-.01 1.08.58 1.23.82.72 1.21 1.87.87 2.33.66.07-.52.28-.87.51-1.07-1.78-.2-3.64-.89-3.64-3.95 0-.87.31-1.59.82-2.15-.08-.2-.36-1.02.08-2.12 0 0 .67-.21 2.2.82.64-.18 1.32-.27 2-.27.68 0 1.36.09 2 .27 1.53-1.04 2.2-.82 2.2-.82.44 1.1.16 1.92.08 2.12.51.56.82 1.27.82 2.15 0 3.07-1.87 3.75-3.65 3.95.29.25.54.73.54 1.48 0 1.07-.01 1.93-.01 2.2 0 .21.15.46.55.38A8.013 8.013 0 0016 8c0-4.42-3.58-8-8-8z"
                  fill="#586069"
                  fill-rule="evenodd"
                  height="22"
                  width="22"
                />
              </g>
            </svg>
          </a>
        </li>
      </ul>
    </div>
  </header>
</div>
`;
